### DISCUSSION TITLE
white matter mask other than provided in the dipy tutorial

### DISCUSSION THREAD

**Discussion by firdausfabrice** [dipy/dipy#2408] [Tractography / Tractometry]
Hello everyone,

I just started exploring the dipy library, starting from the tutorial.
So far I understand every step that is explained, except for the white matter mask part. 
As I tried to use my own data to perform the tractography, I can not use the labels and the white matter mask provided by the example in the tutorial (the one from Stanford), as the dimensions are very different. 

I have tried also just to use a whole brain mask produced using the median_otsu step, however the resulting figure was not satisfactory for a healthy participant (mostly only fibers in the right hemisphere were tracked).

![tractogram_EuDX](https://user-images.githubusercontent.com/86135908/122627145-ff88b100-d0ad-11eb-8316-cbec2d40f0ee.png)

I suspect this mask could be the problem, but I'm aware too that these two problems could be independent of one another, so any help would be much appreciated! 

Thank you

**Comment by skoudoro** [comment]
Hi @firdausfabrice,

Indeed, that's strange. Can you tell us which viewer are you using? 

Also, can you check if your bvecs are correctly oriented? This kind of behavior appears with wrong bvecs in my case (it can be different for you).

**Comment by skoudoro** [comment]
Also, it would great if you share a part of your code to show us how do you seed in the whole brain.
